src/pages/home/report/EmojiPickerMenu/index.native.js

```diff
@@ -29,6 +35,10 @@ class EmojiPickerMenu extends Component {
         this.unfilteredHeaderIndices = [0, 33, 59, 87, 98, 120, 147];
 
         this.renderItem = this.renderItem.bind(this);
+
+        this.state = {
+            emojiSize: {fontSize: dynamicEmojiSize(this.props.windowWidth)},
+        };
```

Comment:
Do you need this state variable? Its not really changing anywhere in the component so I don't quite see the benefit.
How about if you just did,
```
emojiSize={fontSize: dynamicEmojiSize(this.props.windowWidth)}
```
over [here](https://github.com/Expensify/Expensify.cash/pull/3561/files#diff-dc2600e9fa9e3863aa05ccd9f3b3feeb9b282eae5efca59621bee151d0a97388R69).

If you think that's not too readable then we should just do this.emojiSize than save it in a state variable. Thoughts?

Reply:
The idea of keeping it in a state opens the door for any future development for reactivity for emoji sizes.
And adding it to the state doesn't create any overhead.

We can remove it from the state and make it a normal class property.

```diff
- this.state.emojiSize
+ this.emojiSize
```

Reply:
> The idea of keeping it in a state opens the door for any future development for reactivity for emoji sizes.

Making code more complex than it needs to be with the assumption that it will be modified in the future is unnecessary preoptimization. When its needed to be a state variable for any reason then its easy enough to convert so let's not have it be a state variable right now. 

Sure I'm fine with class property 👍  
Could you update?

Reply:
Done I have updated the PR